src/GiGraph.Dot/Extensions/DotGraphExtension.cs

```diff
@@ -97,8 +119,36 @@ public static class DotGraphExtension
         /// <param name="graphGeneratorBuilder">
         ///     The graph generator builder to use in order to get the graph builder to generate the DOT output with.
         /// </param>
+        [Obsolete("Please use the equivalent Build method taking a TextWriter instead of a StreamWriter")]
```

Comment:
So far I never cared about backward compatibility. It would cause me a lot of troubles trying to give the library the current shape. I would just remove the obsolete code ;). Especially that I believe that these pieces of code won't affect anyone.

Reply:
As a first-time user of GiGraph I don't personally care. 😁